executor/README.md

```diff
@@ -0,0 +1,114 @@
+Cook Executor
+=============
+
+The Cook executor is a custom executor written in Python.
+It replaces the default command executor in order to enable a number of features for both operators and end users.
```

Comment:
Should we add something like:

"for more information on Mesos executors, see "Working with Executors" at http://mesos.apache.org/documentation/latest/app-framework-development-guide/"?